powerlibs/aws/process_status_notifier/__init__.py

```diff
@@ -15,8 +15,14 @@ def __init__(self, queue_name, process_name, topics_format='{process_name}__{sta
 
     @cached_property
     def notifier(self):
+        aws_secret_access_key = os.environ.get('AWS_SECRET_KEY') or os.environ.get('AWS_SECRET_ACCESS_KEY')
```

Comment:
I suggest you simply get rid of the `AWS_` preffix. So we would get `REGION`, `SECRET_ACCESS_KEY` and `SECRET_KEY_ID`.
This way (1) the variables meaning would be even more clear and (2) you could write a helper function like `get_aws_env_var("SECRET_KEY_ID")` that handle all this for you.

Reply:
Yup, and this helps getting away from the reserved names for once, as I've already noticed that `AWS_SECRET_KEY` is **also reserved** -_-'